cowprotocol/cowswap
# Add more analytics events for limit orders

# Summary

This PR adds following events to limit orders
```
Send
Sign
SignAndSend
SignToSelf
Reject
Error
```

# To test
- install [GA debug](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en) and turn it on
- open dev-tools and go to console
- go to limit orders page
- try to create an order and reject/approve signing
- you should see these events in console
